hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+# UC5550
+![G400D](images/g400d.jpg)
+# Overview
+---
+The UC5550 is a UCM that supports the STM F4 and F7 processor with 16MB of external RAM. It has a built-on STM WiFi module and an Ethernet PHY. It also has 4MB of external flash for large user programs.
+
+# Specifications
+---
+|					|							|
+| -----------------	| -------------------------	|
+| Processor			| STM 32-bit ARM Cortex-M4	|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+
+# Resources
+---
+## Documents
+* [Schematic](http://files.ghielectronics.com/downloads/Schematics/Systems/G400D%20Dev%20Board%20Schematic.pdf)
+
+## Design Considerations
+This is where we will put the information that the engineer should be aware in order to make their board.  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+
+## Module Pin Configuration  
+| SO-DIMM Pin	| Universal Compute Standard	| Function Name									|
+| :-----------:	| :---------------------------:	| :-------------------------------------------:	|
+| 1				| AGND							| AGND											|
+| 2				| Ethernet TX-					| ETH PHY TX-									|
+| 3				| Bootstrap A					| BOOT											|
+| 4				| Ethernet TX+					| ETH PHY TX+									|
+| 5				| Analog VREF-					| Analog VREF-									|
+| 6				| Ethernet RX-					| ETH PHY RX-									| 
+| 7				| Reserved						| ---------------------------------------------	|
+| 8				| Ethernet RX+					| ETH PHY RX+									|
+| 9				| Reserved						| ---------------------------------------------	|
+| 10			| Ethernet LED SPEED			| ETH PHY LED SPEED								|
+| 11			| Ethernet LED LINK				| ETH PHY LED LINK								|
+| 12			| Reserved						| ---------------------------------------------	|
+| 13			| GND							| GND											|
+| 14			| DCMI D0						| PH9 \| DCMI D0								|
```

Comment:
Use "PH9, DCMI D0, ..." not "PH9 | DCMI D0 | ..."

Reply:
I can do that. I just don't want the commas to cause any confusion.

Reply:
Do any pin names have commas in them?

Reply:
I do not think so but a comma and period look very similar. It could be construed as a typo.

Reply:
We can use a semicolon if it looks clearer.